### ISSUE TITLE
Actual prompt in new line?

### ISSUE THREAD

**Issue by ubuntudroid** [b-ryan/powerline-shell#431]
As the powerline can become quite long it might be beneficial to be able to have the actual input line in a new line below the powerline. Is this already supported?

**Comment by dserodio** [comment]
Yes, there's a `newline` segment

**Comment by ubuntudroid** [comment]
@dserodio perfect, that's exactly what I needed. Didn't find that in documentation though. Is it a hidden segment?

**Comment by dserodio** [comment]
Not _hidden_, just undocumented :) See https://github.com/b-ryan/powerline-shell/tree/master/powerline_shell/segments/ for the authoritative list of segments (assuming your're running the master branch from GitHub instead of installing via pip).

**Comment by b-ryan** [comment]
Yeah I took the list of segments out of the documentation because I was worried about it becoming out of sync with the actual list of segments. I would love to set up some automatic documentation on them.
